wcfsetup/install/files/acp/js/WCF.ACP.js

```diff
@@ -474,9 +474,9 @@ WCF.ACP.Package.Uninstallation = WCF.ACP.Package.Installation.extend({
 		this._elements = elements;
 		this._packageID = 0;
 		
-		if (this._elements !== undefined && this._elements.length) {
+		//if (this._elements !== undefined && this._elements.length) {
```

Comment:
This code (and in line 479, 501) is commented out. You should probably just remove it.